Assets/MixedRealityToolkit/Attributes/obj.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
+guid: fc844539ae188ed479aac171497a0077
```

Comment:
Is this meta file intentional? It looks like an artifact from a generated "obj" folder which was then deleted?